subsys/ipc/ipc_service/backends/Kconfig.icbmsg

```diff
@@ -23,6 +23,14 @@ config IPC_SERVICE_BACKEND_ICBMSG_NUM_EP
 	  backend. The number of endpoints are applied to all the instances,
 	  so this value should be maximum number among all the instances.
 
+config EP_BOUND_WORK_Q_STACK_SIZE
```

Comment:
could you align the name with the Kconfig naming convention in this file?

IPC_SERVICE_BACKEND_ICBMSG_EP_BOUND_WORK_Q_STACK_SIZE